docs/pages/setup/guides/terraform-provider.mdx

```diff
@@ -0,0 +1,236 @@
+---
+title: Terraform Provider
+description: Configuration as a code with Terraform Provider
+---
+
+This section will cover:
+
+- Configuring OpenSSH server `sshd` to join a Teleport cluster. Existing fleets of
+  OpenSSH servers can be configured to accept SSH certificates dynamically issued by a Teleport CA.
+- Configuring OpenSSH client `ssh` to login into nodes inside a Teleport
+  cluster.
+
+## Prerequisites
+
+- [Terraform >= (=terraform.version=)+](https://learn.hashicorp.com/tutorials/terraform/install-cli)
+- Teleport (=teleport.version=) Cloud, Open Source or Enterprise
+
+```bash
+terraform version
+Terraform v(=terraform.version=)
+```
+
+```bash
+tctl version
+Teleport v(=teleport.version=) go(teleport.golang)
+```
+
+Create a folder `teleport-terraform` to hold some temporary files:
+
+```bash
+mkdir -p teleport-terraform
+```
+
+## Step 1/5. Install terraform provider
+
+<Tabs>
+  <TabItem label="Linux">
+  ```bash
+  mkdir -p ${HOME?}/.terraform.d/plugins/gravitational.com/teleport/teleport/(=teleport.version=)/linux_amd64
+  curl -L -O https://get.gravitational.com/terraform-provider-teleport-v(=teleport.version=)-linux-amd64-bin.tar.gz
+  tar -zxvf terraform-provider-teleport-v(=teleport.version=)-linux-amd64-bin.tar.gz -C ${HOME?}/.terraform.d/plugins/gravitational.com/teleport/teleport/(=teleport.version=)/linux_amd64
+  ```
+  </TabItem>
+
+  <TabItem label="MacOS">
+  ```bash
+  mkdir -p ${HOME?}/.terraform.d/plugins/gravitational.com/teleport/teleport/(=teleport.version=)/darwin_amd64
```

Comment:
Missing commands to actually download/unpack terraform?

Reply:
the tar command unpacks to the dir with `-C` flag